.github/workflows/yjit-ubuntu.yml

```diff
@@ -47,6 +47,7 @@ jobs:
         ]
         configure: [
           "--enable-yjit=dev",
+          "--enable-yjit",
```

Comment:
I agree that we're running a lot of tests and we don't necessary need to rerun all the tests in release mode.

I was suggesting we could just test that we are able to run `./configure --enable-yjit` and then build the full ruby, and then just run btest or something. Might be easier to do in a new CI workflow?

Reply:
Yeah, `make check` is btest+test-all+test-spec. So it seems upstream's setup is reasonable. Maybe at some point we want to setup a build farm outside the ruby org so we are not limited to just 6 configs.